cypress/objects/timeLoginPage.js

```diff
@@ -0,0 +1,38 @@
+class TimeLoginPage {
+    visit() {
+        cy.visit('/')
+    }
+
+    getURL() {
+        return cy.url().should('include', '/time-logging')
+    }
+
+    getPageTitle() {
+        return cy.get('.page__title').contains('Timesheets')
+    }
+
+    getCalendarVisable() {
+        return cy.get('.calendar').should('be.visible')
+    }
+
+    getCalendarToday() {
```

Comment:
Again. you should only get today's date and not immediately assert it, since this is a "get" function, not an "assert" function, right? :) Let's drop the 'contains' part and add it in the test cases instead